### ISSUE TITLE
Char 'ー' unread sometimes

### ISSUE THREAD

**Issue by TheScientist14** [AwesomeTTS/awesometts-anki-addon#203]
Hi,

I've implemented TTS on the fly for every card of my japanese deck. I'm using ASUS neural voices.
It's nearly perfect. The problem I've encountered so far is that, sometimes, a word containing 'ー' will be read as if there wasn't/weren't. It's not consistent and I'm pretty sure it comes from the add-on since, when it didn't work, I tested with the same voice (Nanami) on this site : https://azure.microsoft.com/en-us/services/cognitive-services/text-to-speech/#security

For instance : "バター" (ba-ta-a), "サーフボード" (sa-a-fu-bo-o-do), "スープ" (su-u-pu) and "マンゴー" (ma-n-go-o)
will be read as : "バタ" (ba-ta), "サフボド" (sa-fu-bo-do), "スプ" (su-pu) and "マンゴ" (ma-n-go)

I first thought it was bound to the fact that these word contain only katakatan chars but "デーツ" (de-e-tsu), "ストロベリー" (su-to-ro-be-ri-i), "サーフィン" (sa-a-fi-n), "電気シェーバー" (de-n-ki-she-e-ba-a) are read correctly.

Finally, I want to mention that replacing the word that aren't read correctly with their equivalent in hiragana make it works... sometimes.
I.e. : "バター"     -> "ばたー" : ❌,
"サーフボード" -> "さーふぼーど" : ✅,
"スープ"            -> "すーぷ" : ✅ and
"マンゴー"         -> "まんごー" : ✅

Don't know what to think about that. Could be an encoding problem or something else, or multiple problems, idk.
Tell me if I missed any details.
If you want more examples, I could share my deck or something like that.

**Comment by lucwastiaux** [comment]
thanks for reporting this. please email me awesometts@airpost.net as i'd like to do some testing together using other services.

**Comment by lucwastiaux** [comment]
Alright, I think I found out why it doesn't work.
When I check the html of my field, it shows me :
>> <b><span style="font-weight: 400;">マ</span><span style="font-weight: 400;">ンゴー</span></b>
 for example for "マンゴー". I have no clue why, but it seems like this is not an issue from the add-on but from me when I filled my card, I probably messed up with something when I copy-pasted the words.

If I remove the unnecessary stuff from the html, everything works perfectly.
